plugins/repository-gcs/src/test/java/org/elasticsearch/repositories/gcs/MockStorage.java

```diff
@@ -80,6 +80,11 @@ public Bucket get(String bucket, BucketGetOption... options) {
         }
     }
 
+    @Override
+    public Bucket lockRetentionPolicy(final BucketInfo bucket, final BucketTargetOption... options) {
```

Comment:
API change in the dependency made overriding this necessary.
Since we don't use it yet, I added just a dummy override.